model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaMapStorageProviderFactory.java

```diff
@@ -420,9 +420,6 @@ protected EntityManagerFactory createEntityManagerFactory() {
             logger.warnf("Database %s used without lockTimeout option configured. This can result in deadlock where one connection waits for a pessimistic write lock forever.", databaseShortName);
         }
 
-        // Pass on the property to 'EventListenerIntegrator' to activate the necessary event listeners for JPA map storage
-        properties.put(EventListenerIntegrator.JPA_MAP_STORAGE_ENABLED, Boolean.TRUE.toString());
-
```

Comment:
I suppose this would work for Quarkus, but not for Undertow. So could we please leave this in place for now - maybe add a comment to remove it later?

Reply:
@ahus1 This change is already in main :)

IMO, we should think about something different for Undertow.